webservices/resources/committees.py

```diff
@@ -246,14 +246,8 @@ def get_committee(self, args, page_num, per_page, committee_id, candidate_id):
                 else:
                     committees = committees.filter(getattr(Committee, argname)==args[argname])
 
-        # default year filtering
-        if args.get('year') is None:
-            earliest_year = int(sorted(default_year().split(','))[0])
-            # still going or expired after the earliest year we are looking for
-            committees = committees.filter(or_(extract('year', CommitteeDetail.expire_date) >= earliest_year, CommitteeDetail.expire_date == None))
-
         # Should this handle a list of years to make it consistent with /candidate ?
-        elif args.get('year') and args['year'] != '*':
+        if args.get('year') and args['year'] != '*':
```

Comment:
Oh, I remember my thought process on that. Since other branches worked like that, I did not want the user to get errors if they tried, year=* , even though it was unnecessary. I can still take it out though.